tools/AppInsights/Projects/AppInsights_Win10-UAP/ApplicationInsights/Project.json

```diff
@@ -1,14 +1,14 @@
 ﻿{
   "dependencies": {
-    "NuGet.Build.Packaging": "0.1.186"
+    "NuGet.Build.Packaging": "0.2.0"
   },
   "frameworks": {
-    "uap10.0": {}
+    "UAP,Version=v10.0": {}
```

Comment:
this does not seem necessary

Reply:
i meant the entire change from `uap10.0` to `UAP,Version=v10.0`... the FILE seems necessary (!!!!)

---
In reply to: [141440846](https://github.com/Microsoft/WinObjC/pull/2799#discussion_r141440846) [](ancestors = 141440846)